ext/RastersMakieExt/plotrecipes.jl

```diff
@@ -295,19 +295,22 @@ function Makie.convert_arguments(t::Makie.PointBased, A::AbstractRaster{<:Number
     return Makie.convert_arguments(t, _prepare_dimarray(A))
 end
 @static if isdefined(Makie, :SurfaceLike)
-
-    function Makie.convert_arguments(t::SurfaceLike, A::AbstractRaster{var"#s115", 2, D}) where {var"#s115", D<:Tuple}
+    function Makie.convert_arguments(t::SurfaceLike, A::AbstractRaster{T, 2, D}) where {T, D<:Tuple}
         return Makie.convert_arguments(t, _prepare_dimarray(A))
     end
 else # surfacelike is not a thing
+    function Makie.convert_arguments(t::Makie.VertexGrid, A::AbstractRaster{T, 2, D}) where {T, D<:Tuple}
```

Comment:
@asinghvi17 this is not working. So, probably is wrong.

Reply:
what do you mean?  It works fine for me...at least in imagelike

Reply:
The issue is in DD

Reply:
It's this line

https://github.com/rafaqz/DimensionalData.jl/blob/0750b583d3ef3963c96a810e76d76a273008ac08/ext/DimensionalDataMakie.jl#L435

Reply:
so, we need a fix in DD? mmm.